ydb/src/grpc_wrapper/raw_errors.rs

```diff
@@ -20,6 +21,15 @@ impl RawError {
     }
 }
 
+impl<T> From<tokio::sync::mpsc::error::SendError<T>> for RawError {
+    fn from(value: tokio::sync::mpsc::error::SendError<T>) -> Self {
+        Self::Custom(format!(
+            "Error while sending message into gppc stream via mpsc channel: {}",
```

Comment:
Error tokio::sync::mpsc::error::SendError may be throw from different places, not only for grpc.

It need more abstract text for prevent confuce if send error will auto-convert to RawError in other place.